nmpolicy/internal/resolver/replace.go

```diff
@@ -61,7 +61,7 @@ func (r replaceOpVisitor) visitMap(p path, mapToVisit map[string]interface{}) (i
 	}
 	interfaceToVisit, ok := mapToVisit[*p.currentStep.Identity]
 	if !ok {
-		return nil, nil
+		interfaceToVisit = map[string]interface{}{}
```

Comment:
Could we `return map[string]interface{}{}, nil` ?

Reply:
Nope, the recursion will add the next "not found" entrie from the path, we should pass the empty state.

Reply:
Thanks, understood.
I think we can proceed with this solution then. For the current use-case of setting lldp.enabled, we can assume the lldp map to exist in the current state.
If there's a need to set map that doesn't exist, we can re-iterace on https://github.com/nmstate/nmpolicy/pull/107